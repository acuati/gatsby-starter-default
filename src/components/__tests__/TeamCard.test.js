import React from 'react';
import { render } from '@testing-library/react';
import TeamCard from '../teamcard/teamcard';

describe('TeamCard', () => {
  it('renderizamos bien el componente', () => {
    const { container } = render(
      <TeamCard
        imageSrc="https://via.placeholder.com/150"
        title="Sample Title"
        role="jefe de equipo"
        description="me gustan las habichuelas"
      />
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
