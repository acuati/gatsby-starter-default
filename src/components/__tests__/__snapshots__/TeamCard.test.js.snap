// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeamCard renderizamos bien el componente 1`] = `
<div
  class="card-team"
  data-testid="card-team"
>
  <img
    alt="Sample Title"
    class="card-image"
    src="https://via.placeholder.com/150"
  />
  <h2
    class="card-title"
  >
    Sample Title
  </h2>
  <h2
    class="card-role"
  >
    jefe de equipo
  </h2>
  <p
    class="card-description"
  >
    me gustan las habichuelas
  </p>
</div>
`;
